internal/config/config.go

```diff
@@ -102,7 +102,9 @@ type Reporters struct {
 // Tunnel represents a sauce labs tunnel.
 type Tunnel struct {
 	ID     string `yaml:"id,omitempty" json:"id,omitempty"`
```

Comment:
Could you also mark these fields as deprecated via the go doc?
You can do it like this:
```
	// ID represents the tunnel identifier (aka tunnel name).
	// Deprecated. Use Name instead.
	ID     string `yaml:"id,omitempty" json:"id,omitempty"`
```

That way, IDEs will also properly mark this field as deprecated as well.

Reply:
sure!